### ISSUE TITLE
Password support?

### ISSUE THREAD

**Issue by indynz** [andreacfm/Redis-Cache-Extension#9]
Hi Andrea

Is there any way to pass in password while configuring the cache in Railo Admin?

My redis server is setup with the following:

requirepass foobared

How can I pass in the password in the cache settings?

Thanks

**Comment by andreacfm** [comment]
@indynagpal Not really. When I implemented this my use case was using redis in a private network.
Btw the change should not be that hard. May you try to propose a PR?
